index.html

```diff
@@ -0,0 +1,119 @@
+
+    <title>Apple</title>
+</head>
+
+<body>
+    <main class="main-container">
+        <div class="background">
+            <nav class="nav-bar nav-gradiant">
+                <div class="nav-elements apple-logo">
+                    <a href="">
+                        <img src="resources/apple-logo.svg" alt="apple logo">
+                    </a>
+                </div>
+                <div class="nav-elements">
+                    <a class="hover" href="">
+                        <span>Store</span>
+                    </a>
+                </div>
+                <div class="nav-elements">
+                    <a href="">
+                        <span>Mac</span>
+                    </a>
+                </div>
+                <div class="nav-elements">
+                    <a href="">
+                        <span>iPod</span>
+                    </a>
+                </div>
+                <div class="nav-elements">
+                    <a href="">
+                        <span>iPhone</span>
+                    </a>
+                </div>
+                <div class="nav-elements">
+                    <a href="">
+                        <span>iPad</span>
+                    </a>
+                </div>
+                <div class="nav-elements">
+                    <a href="">
+                        <span>iTunes</span>
+                    </a>
+                </div>
+                <div class="nav-elements">
+                    <a href="">
+                        <span>Support</span>
+                    </a>
+                </div>
+                <div class="nav-elements search-img">
+                    <img src="resources/search.svg" alt="">
+                    <input class="nav-search" type="text">
+                </div>
+
+            </nav>
+            <div class="content">
+                <h1>
+                    What will your verbse be?
+                </h1>
+                <h3>
+                    Everyone has something to share. See how prople are using iPad in there unique way to add to the world's story
+                </h3>
+                <div class="play">
+                    <a href="">
+                        Watch the "Tour Verse" film and explore the stories
+                    </a>
+                    <img src="resources/play.png" alt="play">
+                </div>
+
+            </div>
+        </div>
+        <div class="bottom">
+            <div class="bottom-pic1">
+                <img src="resources/5s.jpg" alt="iPhone 5s">
+            </div>
+            <div class="bottom-pic2">
+                <img src="resources/5c.jpg" alt="iPhone 5c">
+            </div>
+            <div class="bottom-pic3">
+                <img src="resources/bday.jpg" alt="Apple Anniversary">
+            </div>
+            <div class="bottom-pic4">
+                <img src="resources/supplier.jpg" alt="Apple Supplier">
+            </div>
```

Comment:
- [ ] This images should be enclosed inside `<a></a>` tags. You can link to '#' if you want.

- [ ] In my screen they are not properly aligned with the background. I will attatch a screenshot.

> ![image](https://user-images.githubusercontent.com/57414789/81759931-ab170900-948b-11ea-8428-6e48420f6ce7.png)